lib/simulator-xcode-9.js

```diff
@@ -579,9 +579,10 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     }
 
     if (globalPrefs.AppleLanguages) {
-      // In order to translate com.apple.SpringBoard, system prompts
-      // for push notification and system prompts for location
-      await B.all(['com.apple.SpringBoard', 'com.apple.locationd']
+      // com.apple.SpringBoard: translates com.apple.SpringBoard and system prompts for push notification
+      // com.apple.locationd: translates system prompts for location
+      // com.apple.tccd: translates system prompts for camera, microphone, contact, photos and app tracking transparency
+      await B.all(['com.apple.SpringBoard', 'com.apple.locationd', 'com.apple.tccd']
```

Comment:
it probably makes sense to move this array into a constant together with the comment

Reply:
thank you! I've modified as https://github.com/appium/appium-ios-simulator/pull/400/commits/549e148380e31939503738cf29895d37e9d957b9